server/routes/client.js

```diff
@@ -17,9 +18,14 @@ router.post('/add', function(req, res, next) {
   client.addUrls(req.body, ajaxUtil.getResponseFn(res));
 });
 
-router.post('/add-files', upload.array('torrents'), function(req, res, next) {
-  client.addFiles(req, ajaxUtil.getResponseFn(res));
-});
+router.post(
+  '/add-files',
+  upload.array('torrents'),
+  booleanCoerce('isBasePath'),
```

Comment:
do you really need to make this a middleware?

Why not
``` js
if (req.body) {
  req.body.isBasePath = req.body.isBasePath === 'true'
}
```

Reply:
That's very specific — I like the middleware approach because it's easy to use elsewhere.

Reply:
I wouldn't think this is a common problem.
Why is it happening in the first place?

Reply:
This is not common, that's true! 

This is happening because a [`FormData`](https://developer.mozilla.org/en-US/docs/Web/API/FormData) instance is used to upload the binary torrent files, and that `FormData` instance also includes the options (one of the options is this `isBasePath`), and `FormData.append` stringifies boolean values.

Reply:
In that event,  I would be specific,  this is clearly an edge-case?

Reply:
It would happen any time `FormData` is used, so I think this should be flexible.

What do you think is the harm in creating and using this middleware?

Reply:
More places to look when debugging!
Do we need more `FormData` instances?

It is a bike-shed at this point,  if you prefer it,  leave it.

Reply:
Fair point, and no, we don't need more `FormData` instances yet.

I do prefer it, so I'll just merge as-is, but I do appreciate your feedback!